index.html

```diff
@@ -27,46 +27,226 @@
+      <div class="about-page">
+        <div class="me">
+          <h1>About Myself</h1>
+          <p class="about-para">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="social">
+          <div class="social-text about-connect"><h4>LET'S CONNECT</h4></div>
+          <div class="icons">
+            <ul>
+              <li>
+                <a href="https://github.com/QueenterJuma"
+                  ><i class="fa-brands fa-github"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/queenter-juma-54a0b1222/"
+                  ><i class="fa-brands fa-linkedin"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://twitter.com/QueenJumah"
+                  ><i class="fa-brands fa-twitter"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-solid fa-m"></i></a>
+              </li>
+              <li>
+                <a href="#"><img src="./image/vector.png" alt="" /></a>
+              </li>
+            </ul>
+          </div>
+          <button>Get my resume</button>
+        </div>
+
+        <div class="language">
+          <div class="dash">
+            <p class="frame">Languages</p>
             <li>
-              <a href="https://www.linkedin.com/in/queenter-juma-54a0b1222/"
-                ><i class="fa-brands fa-linkedin"></i
-              ></a>
+              <span class="fa fa-chevron-down"></span>
             </li>
-            <li>
-              <a href="https://twitter.com/QueenJumah"
-                ><i class="fa-brands fa-twitter"></i
-              ></a>
+          </div>
+          <ul class="lang">
+            <li class="java">
+              <img src="./image/js.png" alt="js logo" />
+              <p>Javascript</p>
             </li>
-            <li>
-              <a href="#"><i class="fa-solid fa-m"></i></a>
+            <li class="java">
+              <img src="./image/html.png" alt="html logo" />
+              <p>HTML</p>
             </li>
-            <li>
-              <a href="#"><img src="./image/vector.png" alt="" /></a>
+            <li class="java">
+              <img src="./image/css.png" alt="css logo" />
+              <p>CSS</p>
             </li>
           </ul>
+          <div class="skill">
+            <div class="arro">
+              <p class="frame">Frameworks</p>
+              <li class="fa fa-chevron-right"></li>
+            </div>
+            <div class="hr-line"><hr /></div>
+            <div class="arro">
+              <p class="frame">Skill</p>
+              <li class="fa fa-chevron-right"></li>
+            </div>
```

Comment:
- [ ] **Good job on making a good design 🥇 🥇** However, It would be better to list all the technology categories ```("Languages", "Frameworks" and "Skills")``` in a ```<ul>``` and each of them in a ```<li>```  😎

Remember, Using multi-level lists in HTML ```(using nested <ul> or <ol> elements)``` allows for the creation of hierarchically structured content, making it easier for users to understand the organization of the information and for search engines to understand the relationships between the different pieces of content.